mlcp/src/main/java/com/marklogic/contentpump/ContentPump.java

```diff
@@ -167,7 +163,10 @@ public static int runCommand(String[] args) throws IOException {
             LOG.error("Cannot run in distributed mode.  HADOOP_CONF_DIR is "
                     + "not configured.");
         }
-        
+        if (distributed) {
+            LOG.warn("MLCP distributed mode has been disabled. Switching to local mode.");
+            distributed = false;
+        }
```

Comment:
I recommend erroring out and exiting instead of automatically switching to local.

Reply:
Okay thanks, I will fix it